docs/internal_api/authentication.rst

```diff
@@ -39,4 +39,20 @@ session information then Publisher will get internal user (or create one if not
 
 .. note::
 
-    Publisher token will be this same as the one from superdesk (provided in :code:`/api/v1/auth/superdesk` request).
\ No newline at end of file
+    Publisher token will be this same as the one from superdesk (provided in :code:`/api/v1/auth/superdesk` request).
+
+Generate Authentication URL for Livesite Editor
+-----------------------------------------------
+
+You can create with API special authentication URL for tenant website. To do that you need to call :code:`/api/v1/livesite/auth/livesite_editor`
+as authorized user (with token in request header or url).
+
+.. code-block:: bash
+
+    curl 'http://publisher.dev/api/v1/livesite/auth/livesite_editor/livesite_editor' -H 'Authorization: d6O3UorCHZ2Pd8PRs/0aXGg1qnT0bKUPWW43dgKqYm3CI4U4Og==' --compressed
```

Comment:
`livesite_editor` is duplicated, shouldnt this be removed according to the route definition?